### PR TITLE
support php7

### PR DESCRIPTION
**Pull request by maherelgamil**
support php7 in composer

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": "~5.4"
+        "php": "~5.4 || ^7.0"
```
Comment (umpirsky) [inline composer.json]: Better: `"php": ">=5.4"`

Comment (maherelgamil) [inline composer.json]: @umpirsky  Yes, your solution is better.

Comment (umpirsky) [inline composer.json]: @maherelgamil I will merge once it is updated. ;)

**Comment by MidnightDesign** [comment]
Can someone update, merge and tag this please?

**Comment by umpirsky** [comment]
@MidnightDesign Please do, it will be quicker. I'll merge new PR and tag. Thanks!

**Comment by MidnightDesign** [comment]
Wait a second. This has already been fixed in #53. I even asked you to merge it and you did. So nevermind. : )

**Comment by umpirsky** [comment]
LoL!

**Pull request closed without merging**